pkg/autodiscovery/listeners/kube_services.go

```diff
@@ -13,7 +13,7 @@ import (
 	"sort"
 	"sync"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Why the explicit `v1` alias?

Reply:
actually goimports adds the alias 
```
$ goimports -d ./pkg/autodiscovery/listeners/kube_services.go
diff -u ./pkg/autodiscovery/listeners/kube_services.go.orig ./pkg/autodiscovery/listeners/kube_services.go
--- ./pkg/autodiscovery/listeners/kube_services.go.orig	2019-03-06 11:25:39.000000000 +0100
+++ ./pkg/autodiscovery/listeners/kube_services.go	2019-03-06 11:25:39.000000000 +0100
@@ -13,7 +13,7 @@
 	"sort"
 	"sync"

-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
but I think it would be better to keep this change for the listeners PR